openfisca_uk/variables/finance/benefit/personal/JSA_contrib.py

```diff
@@ -6,7 +6,7 @@
 class JSA_contrib(Variable):
     value_type = float
     entity = Person
-    label = u"Job Seeker's Allowance (contribution-based)"
+    label = u"JSA (contribution-based)"
```

Comment:
Inconsistent capitalization - is this for it to fit in the PolicyEngine graphs? It'd be good to have some variable metadata that spells out programs IMO.